makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,9 @@
+ALIAS = linux phynode
```

Comment:
this is unused as far as I can see?

Reply:
yes, leftover from some earlier playing around... Seems like i forgot to delete it.